CentOS-2nics-lb-cluster/azuredeploy.json

```diff
@@ -0,0 +1,534 @@
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username for VM"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password for VM"
+            }
+        },
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique storage account name"
+            }
+        },
+        "numberOfInstances": {
+            "type": "int",
+            "defaultValue": 2,
+            "allowedValues": [2,3,4,5,6,7,8,9,10],
+            "metadata": {
+                "description": "Number of VMs to deploy (2-10)"
+            }                
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D2",
+            "allowedValues": [
+                "Standard_A3",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14"
+                ],
+            "metadata": {
+                "description": "Size of the Virtual Machine."
+            }
+        },
+        "imagePublisher": {
+            "type": "string",
+            "defaultValue": "OpenLogic",
+            "metadata": {
+                 "description": "Image Publisher"
+            }
+        },
+        "imageOffer": {
+            "type": "string",
+            "defaultValue": "CentOS",
+            "metadata": {
+                "description": "Image Offer"
+            }
+        },
+        "imageSKU": {
+            "type": "string",
+            "defaultValue": "6.5",
+            "metadata": {
+                "description": "Image SKU"
+            }
+        },
+        "publicIPAddressName": {
+            "type": "string",
+            "defaultValue": "myPublicIP",
+            "metadata": {
+              "description": "Public IP Name"
+            }
+        },
+        "dnsNameforLBIP": {
+            "type": "string",
+            "metadata": {
+              "description": "DNS name for Load Balancer IP"
+            }
+        },
+        "customScriptFilePath": {
+            "type": "string",
+            "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/Centos-2nics-lb-cluster/deploy.sh",
+            "metadata": {
+                "description": "GitHub or blob location of the custom script that configures the cluster"
+            }
+        },
+        "customScriptCommandToExecute": {
```

Comment:
Done as suggested.
G.


On May 28, 2015, at 12:48 PM, Kay Singh <notifications@github.com<mailto:notifications@github.com>> wrote:


In CentOS-2nics-lb-cluster/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/287#discussion_r31270506>:

> +            }
> +        },
> +        "dnsNameforLBIP": {
> +            "type": "string",
> +            "metadata": {
> +              "description": "DNS name for Load Balancer IP"
> +            }
> +        },
> +        "customScriptFilePath": {
> +            "type": "string",
> +            "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/Centos-2nics-lb-cluster/deploy.sh",
> +            "metadata": {
> +                "description": "GitHub or blob location of the custom script that configures the cluster"
> +            }
> +        },
> +        "customScriptCommandToExecute": {


This one as well

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/287/files#r31270506>.